client/mds3/skewer.render.js

```diff
@@ -125,28 +127,35 @@ export function skewer_make(tk, block) {
 		.each(function (d) {
 			d.g = this
 		})
-	// actual disc
-	const discdot = discg.append('circle')
-	// full filled
-	discdot
-		.filter(d => d.dt == dtsnvindel || d.dt == dtsv || d.dt == dtfusionrna)
-		.attr('fill', d => tk.color4disc(d.mlst[0]))
-		.attr('stroke', 'white')
-		.attr('r', d => d.radius - 0.5)
-	// masking half
-	discg
-		.filter(d => d.dt == dtfusionrna || d.dt == dtsv)
-		.append('path')
-		.attr('fill', 'white')
-		.attr('stroke', 'none')
-		.attr('d', d =>
-			d3arc()({
-				innerRadius: 0,
-				outerRadius: d.radius - 2,
-				startAngle: d.useNterm ? 0 : Math.PI,
-				endAngle: d.useNterm ? Math.PI : Math.PI * 2
-			})
-		)
+	const isEmptyShape = tk.skewer?.shape?.[1]?.isFilled == false
```

Comment:
please comment on purpose of `[1]` and `[0]` from skewer.shape

Reply:
The variant label script passes the object key (e.g. 'filledTriangle') as 0 and the object with the .path, .calculatedPath(), and .isFilled as 1. I use the key[0] to determine whether or not use a function or return certain arguments and the properties in the object[1] for rendering. 

I created isEmptyShape to keep the code a little cleaner. It's not necessary, just easier to read.

Reply:
using .shape[0] is okay for quick prototyping, but `.shape={name, ...}` is more understandable and maintainable

Reply:
That's a valid point. I can make that change.